src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/UnusedLocalVariableCheckTest.java

```diff
@@ -241,6 +241,15 @@ public void testUnusedLocalVarNestedClasses3() throws Exception {
                 expected);
     }
 
+    @Test
+    public void testUnusedLocalVarNestedClasses4() throws Exception {
```

Comment:
@shubh220922 if different severity is what killed mutation, why is method naming, input code, input file name focused on nested classes? We should make it obvious what this test does, I would even include a comment stating why we have this test. `testWarningSeverity`, `Input…WarningSeverity`, etc

Reply:
@nrmancuso 

Yes. I have refactored some things. 

We need input code that has violation in its code. We then change ``severity`` to ``warning`` for that to not show any error while running the test. If we hardcode the mutation in ``AbstractViolationReporter.java`` by removing assignment to ``severityLevel`` then ``severityLevel`` will contain the default value which is ``error``. Hence, if input file does not contain any violation that we suppress by mentioning ``severity`` as ``warning`` then the mutation will not be killed.